src/DurableTask.Netherite/OrchestrationService/OrchestrationMessageBatch.cs

```diff
@@ -184,10 +207,11 @@ bool IsExecutableInstance(TaskOrchestrationWorkItem workItem, out string message
                 workItem.OrchestrationRuntimeState.OrchestrationStatus != OrchestrationStatus.Pending)
             {
                 message = $"Instance is {workItem.OrchestrationRuntimeState.OrchestrationStatus}";
+                isNormalSituationForTimerFired = true;
                 return false;
             }
 
-            message = null;
+            isNormalSituationForTimerFired = default;
```

Comment:
nit - same feedback regarding `default` and booleans

Reply:
same answer. We are returning true, so the value of the out parameter is irrelevant.